test/normalization/test_layernorm2d_fp16.cpp

```diff
@@ -2,28 +2,44 @@
 // Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
 
 #include "gtest/gtest.h"
-#include "test_layernorm2d_util.hpp"
+#include "profiler/include/profile_layernorm_impl.hpp"
 
-template <ck::index_t N>
-using I = ck::Number<N>;
+using F16 = ck::half_t;
+using F32 = float;
+using ck::index_t;
 
 template <typename Tuple>
-class TestLayernorm2dFP16 : public ck::TestLayernorm2d<Tuple>
+class TestLayernorm2d : public ::testing::Test
 {
+    protected:
+    using XDataType     = std::tuple_element_t<0, Tuple>;
+    using GammaDataType = std::tuple_element_t<1, Tuple>;
+    using BetaDataType  = std::tuple_element_t<2, Tuple>;
+    using AccDataType   = std::tuple_element_t<3, Tuple>;
+    using YDataType     = std::tuple_element_t<4, Tuple>;
+
+    void Run()
+    {
+        // N, H, W, G, C
+        std::vector<std::vector<ck::index_t>> lengths = {
+            {4, 256}, {8, 511}, {9, 1032}, {4, 2048}, {1, 8192}, {4000, 2000}};
```

Comment:
Question: for a single case, ```{4, 256}```, which number is ```N H W G C```?

Reply:
It is just 2D. I fix the comment in 33d760fc8adae96b3134a63516f282beba279aab